massa-node/src/main.rs

```diff
@@ -505,7 +518,13 @@ async fn launch(
         protocol: ProtocolCommandSender(protocol_command_sender.clone()),
         storage: shared_storage.clone(),
     };
-    let factory_manager = start_factory(factory_config, node_wallet.clone(), factory_channels);
+    let factory_manager = start_factory(
+        factory_config,
+        node_wallet.clone(),
+        factory_channels,
+        denunciation_factory_receiver,
```

Comment:
maybe they can be put in FactoryChannels ?

Reply:
Good idea. I've added a note to do that in a follow up :)